RELEASE-NOTES.rst

```diff
@@ -2,7 +2,7 @@
  Invenio-Assets v1.0.0
 =======================
 
-Invenio-Assets v1.0.0 was released on December 4, 2015.
+Invenio-Assets v1.0.0 was released on June 29, 2016.
 
```

Comment:
Well, at least it looks the same as all other packages so find&replace script work out-of-box 😉